packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue

```diff
@@ -115,6 +121,9 @@ const isRunning = (item: NotificationItem) => item.status === ProgressState.Runn
 const isCancelling = (item: NotificationItem) => item.status === ProgressState.Cancelling;
 const isCancelled = (item: NotificationItem) => item.status === ProgressState.Cancelled;
 
+const getProjectName = (item: NotificationItem) =>
+	(useProjects().allProjects.value || []).find((p) => p.id === item.projectId)?.name || '';
+
```

Comment:
Maybe we should create a map in the project composable so we don't have to do linear look up for every event-group.